### ISSUE TITLE
Info button next to "added to contacts" notification does not have alt text and button labeling

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#559]
When using VoiceOver or Talkback, when focused on the info button of the Contacts notification in a proof request or credential offer, the reader is silent. The button requires alt text and button labeling

When focused on the info button, it should read "What are contacts? button"

![image.png](https://images.zenhubusercontent.com/619bd30a068b26108b1bc2fe/9ed851d1-1a38-4d10-893d-6d4c3723d2ff)
